service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/usecase/DeleteTaxiByIdUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package org.thechance.service_taxi.domain.usecase
+
+interface DeleteTaxiByIdUseCase {
```

Comment:
I think there is no need for` DeleteTaxi**ById**UseCaseit` would be better to be like just `DeleteTaxiUseCase`